### PR TITLE
fix #66135 iphone-mania.jp

### LINKED ISSUE #66135: iphone-mania.jp
### Issue URL (Ads)
 [https://iphone-mania.jp/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fiphone-mania.jp%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.12
Browser: | EdgeChromium
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Japanese

### PR DESCRIPTION
**Pull request by Yuki2718**
https://github.com/AdguardTeam/AdguardFilters/issues/66135

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -182,6 +182,8 @@ punyu.com##.box_title_pr + .rank_table
 punyu.com##.box_title[style="width:330px; margin-top:10px;"]
 punyu.com##.tblmain[cellspacing="2"][cellpadding="0"]
 punyu.com##div[id^=dugaflvplayer-]
+iphone-mania.jp###jack
```
Comment (zloyden) [inline JapaneseFilter/sections/specific.txt:185]: I suggest to change it to `iphone-mania.jp##div[id^="jack"]`. If you can't make a change, please let me know.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:185]: That causes FP.

Comment (zloyden) [reply JapaneseFilter/sections/specific.txt:185]: Hm, I guess you meant incorrect blocking. What about this `iphone-mania.jp###jack_wrapper > div[id^="jack"]` one?

Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:185]: Looks okay, will change.

Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt]: Done.

**Review by zloyden — APPROVED** [review]

**Pull request merged**